### ISSUE TITLE
Printing Copyright text optionally

### ISSUE THREAD

**Issue by soimsoim** [amzn/askalono#54]
This seems to find the license only in the representative license file (ex- NOTICE, LICENSE.txt). Is that right?
At this time, it would be really nice to print copyright text in the file as an execution option. 
Is there a way to do this?

Best regards

**Comment by jpeddicord** [comment]
That is correct. In fact, askalono actively tries to throw out copyright data when identifying a license, as it helps with license matching and speed.

It would be possible to add a mode to preserve and extract copyrights, but a significant overhaul would be needed to some components. I'm open to the possibility.

Right now, if you want to extract copyright data, I recommend checking out https://github.com/nexB/scancode-toolkit.

**Comment by sschuberth** [comment]
Just chiming in here as an [ORT](https://github.com/oss-review-toolkit/ort) maintainer to say that it would be awesome if if askalono would maintain / also detect copyright statements, as ScanCode has various issues in that regard (like tampering with casing and not maintaining special characters like German Umlauts).
